plutus-conformance/src/Common.hs

```diff
@@ -1 +1,68 @@
-module PlutusConformance.Common where
+{-# LANGUAGE OverloadedStrings #-}
+
+module Common where
+
+import GHC.IO (unsafePerformIO)
+import PlutusCore.Core (Version (Version), defaultVersion)
+import PlutusCore.Default (DefaultFun, DefaultUni)
+import PlutusCore.Evaluation.Machine.ExBudgetingDefaults (defaultCekParameters)
+import PlutusCore.Evaluation.Result (EvaluationResult)
+import PlutusCore.Name (Name)
+import Prelude
+import Test.Tasty
+import Test.Tasty.HUnit
+import Text.Megaparsec (SourcePos)
+import UntypedPlutusCore.Core.Type qualified as UPLC
+import UntypedPlutusCore.Evaluation.Machine.Cek (unsafeEvaluateCekNoEmit)
+
+data TestContent =
+    MkTestContent {
+        testName    :: FilePath
+        , expected  :: EvaluationResult UplcProg
+        , inputProg :: UplcProg
+    }
+
+mkTestContents :: [FilePath] -> [EvaluationResult UplcProg] -> [UplcProg] -> [TestContent]
+mkTestContents lFilepaths lRes lProgs =
+    if length lFilepaths == length lRes && length  lRes == length lProgs then
+        [MkTestContent file res prog | file <- lFilepaths, res <- lRes, prog <- lProgs]
```

Comment:
But this is not parallel list comprehensions and I assume you wanted zipping here given all the `length` checks? Seems like a bug.